mkdocs.yml

```diff
@@ -94,6 +94,8 @@ nav:
     - Tool Integration:
       - NPM Integration Example: user-guide/tool-integration/npm-example.md
       - SBT Integration Example: user-guide/tool-integration/sbt-example.md
+      - MAVEN Example: user-guide/tool-integration/maven-example.md
```

Comment:
No need for these to be all uppercase. Please, change them to `Maven Example` and `Gradle Example`. Thanks!